### DISCUSSION TITLE
HTTP 500, End of Script output error

### DISCUSSION THREAD

**Discussion by 3isenHeiM** [kimai/kimai#3299] [General]
**Describe the bug**
This is a new Kimai installation, and when browsing to the tabs in the "Administration" section, I have HTTP error 500 pages.

![image](https://user-images.githubusercontent.com/26417172/167365298-07bda3ca-8ccc-4972-bbd6-8e9b644c33e9.png)

It's installed on a shared hosting, and when checking the errors (in the provider logs), I see multiple occurrences of the following error : 

````
End of script output before headers: index.php, referer: https://rfs.vivegnis.be/fr/team/timesheet/
End of script output before headers: index.php, referer: https://rfs.vivegnis.be/en/admin/activity/
````

**Logfile**
Nothing is present in the `prod.log` file!

**Additional context**
 - Kimai version: 1.19.5 stable 
 - PHP version: 7.4.25

**Comment by kevinpapst** [comment]
Shitty webhoster... check Kimai logs at Doctor > Logs or at var/log/prod.log

**Reply by 3isenHeiM** [reply]
As mentionned in the issue, nothing about this error is present in the prod.log file.

I don't see why its on the provider side, since only a few tabs return that error.
My provider is OVH, world-famous.

**Reply by kevinpapst** [reply]
Because it works on thousand of installations. Shared hosting does not work well and is known for the most absurd errors.

**Comment by sg6** [comment]
I've had the exact same issue - I know, it's been a few months, but for users googling the error:
Webhoster is (probably) not shitty, just the vendor directory was empty. Run `composer install` (either on a local machine and copy/paste vendor directory or try get composer running on your web host), then it all works.

**Reply by kevinpapst** [reply]
I just can repeat myself: Kimai is not supported/compatible with shared hosting.

Anyway: thanks for sharing your solution 👍
